src/trg-preferences-dialog.c

```diff
@@ -756,7 +756,12 @@ static GtkWidget *trg_prefs_viewPage(TrgPreferencesDialog * dlg)
     hig_workarea_add_wide_control(t, &row, w);
 #endif
 
-	hig_workarea_add_section_title(t, &row, _("System Tray"));
+#if HAVE_LIBAPPINDICATOR
```

Comment:
I generally prefer the approach of keeping the section, but greying it out as insensitive with a tooltip that says the project was built without Appindicator support. It might be slightly technical but it clears up some confusion about which features actually exist. Realistically only users who build it themselves will even encounter this situation and can handle it.

Reply:
Good idea